tests/scoring_test.py

```diff
@@ -0,0 +1,4 @@
+import http_server
+
+def test_best_case_star_voting():
+    pass
```

Comment:
t e m s t

Reply:
yes yes i know i know i need to get around to finishing that